graphql-java-support/src/test/java/com/apollographql/federation/graphqljava/FederationTest.java

```diff
@@ -23,29 +26,42 @@
 
 class FederationTest {
     private final String emptySDL = TestUtils.readResource("schemas/empty.graphql");
+    private final String emptySDLOutput = TestUtils.readResource("schemas/empty-output.graphql");
     private final String interfacesSDL = TestUtils.readResource("schemas/interfaces.graphql");
     private final String isolatedSDL = TestUtils.readResource("schemas/isolated.graphql");
     private final String productSDL = TestUtils.readResource("schemas/product.graphql");
+    private final String productSDLOutput = TestUtils.readResource("schemas/product-output.graphql");
```

Comment:
It wasn't clear to me at first why a separate schema is needed here. I realize now that it's because a few new directives started appearing and I suspect an error occurs when loading the schema with the directives expressed.